README.md

```diff
@@ -10,160 +10,159 @@ Eventum is a content management system for an event-driven blog that syncs with
 - Authentication is done with [Google+ server-side flow][google-plus-server-side-flow]
 - Forms and validation are done through [Flask-WTForms][flask-wtforms] and [WTForms][wtforms]
 - CSS is generated from [SCSS][scss] and managed with [Flask-Assets][flask-assets]
+- Configurations are managed using [Consul][consul].
+- Development is done using [Vagrant][vagrant].
+- Deployment is done using [Docker][docker].
 
-## Setup
+## First-Time Setup
 
-Eventum runs on Linux and OSX.  To get it up an running, follow these steps:
+Eventum can be run locally on your machine, or in a virtual environment, using [Vagrant][vagrant].  Default to use Vagrant, unless you have some reason to run Eventum locally.
 
-1.  Install [MongoDB][mongodb] ([Ubuntu Linux][mongodb-linux], [OSX][mongodb-osx]).
+#### Using Vagrant 
 
-    > On OSX, you may have to run `$ mkdir /data /data/db` before you can run `mongod` without errors.
+1.  Download the [ADI website secrets][adi-website-secrets] from GitHub. Unzip the folder, and copy its **contents** into the `config` directory. 
+    
+    > If the link gives a 404 error, ask someone within the ADI GitHub organization for access
 
-2.  Install [VirtualEnv][virtualenv]:
-    ```bash
-    $ sudo pip install virtualenv
-    ```
+2. Install [Virtual box](https://www.virtualbox.org/wiki/Downloads).
 
-3.  Download a copy of `client_secrets.json` from the [Google Developer Console][google-developer-console] or from a friend, and place it in the `config` folder.
+3. Once Vagrant is installed, install [Vagrant][vagrant-install].
 
-4.  Authorize the Google Calendar API:
-    ```bash
-    $ python authorize.py
-    ```
-5.  Install SASS gem `gem install sass`
-    * otherwise, you will see an `OS` error
+#### Without Vagrant
 
-Then you should be all set to run!
+Eventum runs natively on Linux and OSX.  To get it up an running, follow these steps:
 
-## Developing
+1.  Download the [ADI website secrets][adi-website-secrets] from GitHub. Unzip the folder, and copy its **contents** into the `config` directory.  
```

Comment:
Can we have a default set of configurations so that someone can start up the app w/o access to secrets?